lib/screens/user_registration/widgets/email_input.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+
+class EmailField extends StatelessWidget {
+  const EmailField(this.controller);
+  final TextEditingController controller;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
```

Comment:
no need for this container :)

Reply:
Resolved but Similar fix needs to be done in login branch